credentials/src/main/kotlin/web5/sdk/credentials/VerifiableCredential.kt

```diff
@@ -64,15 +63,15 @@ public class VerifiableCredential internal constructor(public val vcDataModel: V
    * ```
    */
   @JvmOverloads
-  public fun sign(did: Did, assertionMethodId: String? = null): String {
-    val payload = JWTClaimsSet.Builder()
+  public fun sign(did: BearerDid, assertionMethodId: String? = null): String {
+    val payload = JwtClaimsSet.Builder()
       .issuer(vcDataModel.issuer.toString())
-      .issueTime(vcDataModel.issuanceDate)
+      .issueTime(vcDataModel.issuanceDate.time)
       .subject(vcDataModel.credentialSubject.id.toString())
-      .claim("vc", vcDataModel.toMap())
+      .misc("vc", vcDataModel.toMap())
```

Comment:
is `misc` the more official name than `claim`?

Reply:
`misc` is the top level field where we shove everything that's not a reserved keyword in JWT. i've followed the implementation in web5-dart

https://github.com/TBD54566975/web5-dart/blob/main/packages/web5/lib/src/jwt/jwt_claims.dart#L59

@mistermoe is there a spec i can reference for neal / do we need to write one?

Reply:
the standard to use is public / private claim names, depending on whether the claims are registered in an IANA registry

https://datatracker.ietf.org/doc/html/rfc7519#section-4.2

Reply:
I think `claims` is more apt than `misc`

Reply:
leaving it as `misc` for now, but this needs a discussion and a decision - https://github.com/TBD54566975/web5-spec/issues/143

Reply:
agreed @decentralgabe . `misc` is just something i came up with to avoid having to use the terms `private` and `public` in code because those are reserved. i think @nitro-neal 's `claim()` suggestion is a great idea. can address in a subsequent PR